functions/src/create-contract.ts

```diff
@@ -94,58 +71,47 @@ export const createContract = newEndpoint(['POST'], async (req, _res) => {
 
   const userContractsCreatedTodaySnapshot = await firestore
     .collection(`contracts`)
-    .where('creatorId', '==', creator.id)
+    .where('creatorId', '==', user.id)
     .where('createdTime', '>=', freeMarketResetTime)
     .get()
   console.log('free market reset time: ', freeMarketResetTime)
   const isFree = userContractsCreatedTodaySnapshot.size === 0
 
   const ante = FIXED_ANTE
 
-  if (
-    ante === undefined ||
-    ante < MINIMUM_ANTE ||
-    (ante > creator.balance && !isFree) ||
-    isNaN(ante) ||
-    !isFinite(ante)
-  )
-    throw new APIError(400, 'Invalid ante')
-
   console.log(
     'creating contract for',
-    creator.username,
+    user.username,
     'on',
     question,
     'ante:',
     ante || 0
   )
 
   const slug = await getSlug(question)
-
   const contractRef = firestore.collection('contracts').doc()
-
   const contract = getNewContract(
     contractRef.id,
     slug,
-    creator,
+    user,
     question,
     outcomeType,
     description,
-    initialProb,
+    initialProb ?? 0,
     ante,
     closeTime,
     tags ?? [],
     NUMERIC_BUCKET_COUNT,
     min ?? 0,
     max ?? 0,
-    manaLimitPerUser ?? 0
+    0
```

Comment:
I think we should remove the manaLimitPerUser. Most of this feature has already been removed.

Reply:
Then I will!